code/WorkInProgress/JobXPRewards.dm

```diff
@@ -617,3 +617,42 @@ mob/verb/checkrewards()
 		I.set_loc(get_turf(C.mob))
 		C.mob.put_in_hand(I)
 		return
+
+/////////////Chef////////////////
+
+/datum/jobXpReward/chef/ovenmitts
+	name = "Chef's Oven Mitts"
+	desc = "Finally, after years of burning your fingers on baking trays, soup pots and deepfryers, you get your own pair of oven mitts! "
+	required_levels = list("Chef"=0)
```

Comment:
If it's a zero XP reward, why not just put in their spawn inventory?

Reply:
The main goal is to just provide basic Job XP rewards, to add some meat to the bare bones system, so to speak. I'd prefer not to clog player inventory's and it makes the items feel more special to players.